apps/design/backend/src/worker/generate_election_package.ts

```diff
@@ -26,6 +29,41 @@ import { PORT } from '../globals';
 import { WorkerContext } from './context';
 import { createBallotPropsForTemplate } from '../ballots';
 
+export interface V3SystemSettings {
+  readonly auth: SystemSettings['auth'];
+  readonly markThresholds: MarkThresholds;
+  readonly centralScanAdjudicationReasons: readonly AdjudicationReason[];
+  readonly precinctScanAdjudicationReasons: readonly AdjudicationReason[];
+  readonly precinctScanDisallowCastingOvervotes: boolean;
+}
+
+function makeV3Compatible(zip: JsZip, systemSettings: SystemSettings): void {
+  zip.remove(ElectionPackageFileName.METADATA);
+  zip.remove(ElectionPackageFileName.APP_STRINGS);
+  zip.remove(ElectionPackageFileName.AUDIO_IDS);
+  zip.remove(ElectionPackageFileName.AUDIO_CLIPS);
+  zip.remove(ElectionPackageFileName.SYSTEM_SETTINGS);
```

Comment:
What's the timeline to sunset this v3 support? If by the election after next (ie. before we make significant changes to the v4 election package format), this seems fine.

If not, I worry that additions to the zip in `generateElectionPackage` may not be paired with a corresponding `zip.remove` call here. If that's the case, have we considered if it's worth it to factor out zip construction so a function like `makeV3CompatibleZip` could _only_ pick its relevant components rather than building a v4 zip then deleting parts?